contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -3514,6 +3514,25 @@ rewriteBatchLevelStatement(
 				if (all_removed)
 					removeTokenStringFromQuery(expr, cctx->WITH(), ctx);
 			}
+
+			if (cctx->table_type_definition() && cctx->table_type_definition()->column_def_table_constraints())
+			{
+				for (auto cdtctx : cctx->table_type_definition()->column_def_table_constraints()->column_def_table_constraint())
+				{
+					if (cdtctx->table_constraint() && cdtctx->table_constraint()->UNIQUE())
+						rewritten_query_fragment.emplace(std::make_pair(cdtctx->table_constraint()->UNIQUE()->getSymbol()->getStopIndex()+1 , std::make_pair("", " NULLS NOT DISTINCT")));
+
+					if (cdtctx->column_definition() && !cdtctx->column_definition()->column_constraint().empty())
```

Comment:
Cant we just have an enter/exit function for column_constraint?

Reply:
unsure why we are explicitly calling functions like ```post_process_table_constraint``` or ```post_process_column_definition``` , thinking if its related to how our query mutator is implemented ?